test/specs/deployAndRetrieve.e2e.ts

```diff
@@ -290,6 +310,12 @@ describe('Deploy and Retrieve', async () => {
     );
     await browser.keys(['prefer deploy']);
 
+    try {
+      await utilities.runCommandFromCommandPrompt(workbench, 'View: Close Panel', 2);
+    }
+    catch {
+      utilities.log('Panel is already closed');
+    }
```

Comment:
This was added because the setting "Prefer Deploy on Save" is hidden by the Output panel when the VSCode window is too small.